src/core_mapfiles/ecl.rs

```diff
@@ -98,8 +100,8 @@ static ECL_06: &'static CoreSignatures = &CoreSignatures {
         (Th06, 15, Some(("SSS", Some(IKind::BinOp(B::Mul, Ty::Int))))),
         (Th06, 16, Some(("SSS", Some(IKind::BinOp(B::Div, Ty::Int))))),
         (Th06, 17, Some(("SSS", Some(IKind::BinOp(B::Rem, Ty::Int))))),
-        (Th06, 18, Some(("S", None))),
-        (Th06, 19, Some(("S", None))),
+        (Th06, 18, Some(("S", None))), // Some(IKind::UnOp(U::Inc, Ty::Int))
+        (Th06, 19, Some(("S", None))), // Some(IKind::UnOp(U::Dec, Ty::Int))
```

Comment:
as I wrote on Discord, I don't think any syntax can feasibly map to these, because `++X` and `X++` both produce values and thus would need the signature `SS`